bugsnag-android-core/src/main/java/com/bugsnag/android/NativeStackframe.kt

```diff
@@ -59,9 +59,9 @@ class NativeStackframe internal constructor(
         writer.name("method").value(method)
         writer.name("file").value(file)
         writer.name("lineNumber").value(lineNumber)
-        writer.name("frameAddress").value(frameAddress)
-        writer.name("symbolAddress").value(symbolAddress)
-        writer.name("loadAddress").value(loadAddress)
+        frameAddress?.let { writer.name("frameAddress").value("0x%x".format(it)) }
+        symbolAddress?.let { writer.name("symbolAddress").value("0x%x".format(it)) }
+        loadAddress?.let { writer.name("loadAddress").value("0x%x".format(it)) }
```

Comment:
This will need to handle cases where the value came from an unsigned high-value long (where the most-significant bit is set).